virtualbox/machine.go

```diff
@@ -690,13 +690,13 @@ func (m *Machine) Modify() error {
 
 // AddNATPF adds a NAT port forarding rule to the n-th NIC with the given name.
 func (m *Machine) AddNATPF(n int, name string, rule driver.PFRule) error {
-	return vbm("controlvm", m.Name, fmt.Sprintf("natpf%d", n),
+	return vbm("modifyvm", m.Name, fmt.Sprintf("--natpf%d", n),
```

Comment:
For the record, yes, I believed this not to work without this change. That seems consistent with the VirtualBox documentation. After all, this call is being made while the VM is shut down, or is it not?